generative_ai/gemini_safety_config_example.py

```diff
@@ -40,18 +37,6 @@ def generate_text() -> str:
         max_output_tokens=2048, temperature=0.4, top_p=1, top_k=32
     )
 
-    # Safety config
```

Comment:
Safety config stands on the idea to show how safety settings works. If you remove them, the sample won't make sense. 
The request was to remove unrelated stuff few lines above
```
generation_config = GenerationConfig(
        max_output_tokens=2048, temperature=0.4, top_p=1, top_k=32
)
```
Which has nothing to do with safety, but just adds additional unnecessary complexity.
Please roll back these changes and remove **GenerationConfig**

Reply:
Thanks! Got it